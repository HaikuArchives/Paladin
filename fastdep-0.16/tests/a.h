#ifndef A_H_
#define A_H_

#include "c.h"

#endif
