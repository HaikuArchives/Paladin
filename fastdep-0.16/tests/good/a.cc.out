a.o: a.cc \
	a.h \
	c.h \
	d.h \
	b.h \
	e.h
